cmd/plasmacli/cmd/eth/exit.go

```diff
@@ -74,12 +76,12 @@ Transaction Exit Usage:
 			From:     auth.From,
 			Signer:   auth.Signer,
 			GasLimit: gasLimit,
-			Value:    big.NewInt(minExitBond),
```

Comment:
why was this changed? seems harder to update now

Reply:
ah, minExitBond was defined in another file and only used in this location so i just inlined it. I can move it back